modules/loader/src/main/java/com/opengamma/strata/loader/csv/FullSwapTradeCsvLoader.java

```diff
@@ -660,6 +663,23 @@ private static PriceIndexCalculationMethod parseInflationMethod(Optional<String>
         .map(cal -> BusinessDayAdjustment.of(dateCnv, cal));
   }
 
+  // days adjustment, defaulting business day convention and holiday calendar
+  private static BusinessDayAdjustment parseBusinessDayAdjustmentWithFallback(
+      CsvRow row,
+      String leg,
+      String cnvField,
+      String calField) {
+
+    Currency currency = Currency.of(getValueWithFallback(row, leg, CURRENCY_FIELD));
```

Comment:
I don't think this can result in `BusinessDayAdjustment.NONE`. If both fields are empty, `NONE` should be the result.